web_norobots/views/disable_robots.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<openerp>
+    <data>
+        <template id="website.robots">
```

Comment:
No, the only view where website.robots is defined is addons/website/views/website_templates.xml
It is used in addons/website/controllers/main.py to display the robots.txt file in that http.route.
There are no extensions of the website.robots view anywhere but there..